src/sass/variables.sass

```diff
@@ -3,8 +3,9 @@ $base-color-text-grey: #9b9b9b
 $base-color-light-grey: #e7e7e7
 $arrow-color: #E6E6E6
 
-$sm: 320px
-$md: 768px
-$lg: 1024px
-$xl: 1440px
-$xxl: 2560px
\ No newline at end of file
+// Screen resolutions
+$screen-resolution-mobile: 320px
```

Comment:
$mobile, $tablet-portrait, $tablet-landscape, $desktop, $large-desktop

Reply:
Changed to
`@mixin for-tablet-up
  @media (min-width: $tablet-portrait)
    @content`

`@include for-tablet-up
        flex-direction: row
        text-align: center
        padding: 1em`

Reply:
[Done](https://github.com/JuliaKH/arsenal/pull/31/commits/5b524a59b75cff78d5ff70e23e5a1b4e63a9d2c8?file-filters%5B%5D=.css&file-filters%5B%5D=.map&file-filters%5B%5D=.sass#r300268855)